controllers/apply-next/simple/fields.js

```diff
@@ -160,6 +160,10 @@ const allFields = {
             cy: 'TODO'
         },
         type: 'textarea',
+        settings: {
+            showWordCount: true,
+            maxWords: 100
```

Comment:
Should we use this same value in the Joi validation schema too?

Reply:
Quite possibly, but I'm OK leaving it separate for now. Something to explore, my take is that we might want the server limit to be a tad higher. i.e. the word count encourages 500 words, but if you submit 501 we might not stop you.